src/Math-Complex/PMComplex.class.st

```diff
@@ -650,6 +650,11 @@ PMComplex >> sign [
 	^ real sign
 ]
 
+{ #category : #testing }
+PMComplex >> signBit [
```

Comment:
I think, as this is a one line block, we might inline this, 

```
PMComplex >> signBit [ ^ self real signBit ]
```

unless what you've done is the idiomatic way.

Reply:
I don't understand ? This is not a block. You are looking at Tonel files, the real method is : 
MComplex >> signBit
^ self real signBit